README.md

```diff
@@ -33,10 +33,10 @@
   mkdir -p /home/pcloud/data/mysql
   mkdir -p /home/pcloud/data/certs
   ```
-  - Copy Kubernetes CA files (ca.pem, ca-key.pem, ca.srl) to `pcloud_data/certs` folder
-  - Copy Kubernetes admin user key (admin.pem, admin-key.pem) to `pcloud_data/certs` folder
-  - Copy CephFS Key file(admin.secret) to `pcloud_data/certs` folder
-  - Copy `/paddlecloud/settings.py` file to `pcloud_data` folder
+  - Copy Kubernetes CA files (ca.pem, ca-key.pem, ca.srl) to `pcloud/data/certs` folder
```

Comment:
Maybe not put strict folders here, the folders are used by `k8s/cloud_deployment.yaml`, and these folders need to be full path begin from `/`. So users need to define their own hostpath for `yaml` files to use.